buildSrc/src/main/kotlin/otel.java-library-conventions.gradle.kts

```diff
@@ -5,6 +5,7 @@ import ru.vyarus.gradle.plugin.animalsniffer.AnimalSniffer
 plugins {
     id("java-library")
     id("org.jetbrains.kotlin.jvm")
+    id("kotlin-kapt")
```

Comment:
Sure, the android libs need it but do the java libs really also need this?

Reply:
Good point, I've removed it from the java lib conventions.